UI/pre-stream-wizard/page-input-display.cpp

```diff
@@ -0,0 +1,49 @@
+#include "page-input-display.hpp"
+
+#include <QWizardPage>
+#include <QVBoxLayout>
+#include <QScrollArea>
+#include <QFormLayout>
+#include <QLabel>
+
+#include "pre-stream-current-settings.hpp"
+
+QWizardPage *SettingsInputPage(StreamWizard::EncoderSettingsRequest *settings)
+{
+	QWizardPage *page = new QWizardPage();
+	page->setTitle("Demo + Show Data");
+
+	// Layout for the entire widget / Wizard PAge
+	QVBoxLayout *mainLayout = new QVBoxLayout(page);
+
+	// Scroll area that contains values
+	QScrollArea *scroll = new QScrollArea();
+
+	// Data list
+	QWidget *formContainer = new QWidget();
+	QFormLayout *form = new QFormLayout(formContainer);
+
+	form->addRow("Server Url",
+		     new QLabel(QString::fromUtf8(settings->serverUrl)));
```

Comment:
It's possible to use data-table-driven code in this case as well, storing a pointer-to-member in the table and using the ->* operator to resolve to a value.

Reply:
Outside of scope for now but I like it, will come back to that.